content/audio.md

```diff
@@ -144,79 +179,38 @@ If you would like to monitor PipeWire, run:
-systemctl restart --user pipewire.service
-```
-
-This change can be undone using these commands:
-
-```bash
-sudo sed -i 's/\["session.suspend-timeout-seconds"\] = 0/--\["session.suspend-timeout-seconds"\] = 5/' /usr/share/wireplumber/main.lua.d/50-alsa-config.lua
-systemctl restart --user pipewire.service
-```
+## Installing Sound Open Firmware Binaries
 
-#### Prevent Crackling with PulseAudio
+This firmware is essential for proper audio functionality on systems using Intel audio hardware that relies on Sound Open Firmware.
 
-For older versions of Pop!_OS or distributions using PulseAudio without PipeWire, these two commands will disable this behavior and restart PulseAudio:
+To install, run the command:
 
-```bash
-sudo sed -i 's/load-module module-suspend-on-idle/#load-module module-suspend-on-idle/' /etc/pulse/default.pa
-pulseaudio -k
 ```
-
-This change can be undone using these commands:
-
-```bash
-sudo sed -i 's/#load-module module-suspend-on-idle/load-module module-suspend-on-idle/' /etc/pulse/default.pa
-pulseaudio -k
+sudo apt update
+sudo apt upgrade
+sudo apt install firmware-sof-signed
 ```
 
-### PCI/internal sound card not detected (dummy output)
+## Installing Alsa Firmware Loaders
 
-With hardware that uses the `snd_hda_intel` kernel module, rare [bugs](https://bugs.launchpad.net/ubuntu/+source/linux-oem-osp1/+bug/1864061) can cause the sound card to not be detected. If you're having this issue, try running these commands to force the usage of a specific audio driver:
+This package provides firmware files that may be necessary for certain sound cards to function correctly with the Advanced Linux Sound Architecture (ALSA).
 
-```
-echo "options snd-hda-intel dmic_detect=0" | sudo tee -a /etc/modprobe.d/alsa-base.conf
-echo "blacklist snd_soc_skl" | sudo tee -a /etc/modprobe.d/blacklist.conf
-```
-
-Reboot after making the changes. If this doesn't solve the issue, undo the changes using these commands:
+To install, run the command:
 
 ```
-sudo sed -i 's/options snd-hda-intel dmic_detect=0//' /etc/modprobe.d/alsa-base.conf
-sudo sed -i 's/blacklist snd_soc_skl//' /etc/modprobe.d/blacklist.conf
+sudo apt update
+sudo apt upgrade
+sudo apt install alsa-firmware-loaders
 ```
 
-Then reboot again.
-
 ## Gather Information for Support
 
-The `alsa-info` command will gather a number of outputs, including some of the above-listed outputs, and package them so they can be shared easily. In a terminal, run the command:
+The 'alsa-info' command will gather a number of outputs, including some of the above-listed outputs, and package them so they can be shared easily. In a terminal, run the command:
 
 ```
 alsa-info
 ```
 
 When the script is finished gathering information, type <kbd>y</kbd> and press Enter to upload your output to the ALSA website, then send the link to Support.
 
-![alsa-info](/images/audio/alsa-info.png)
+![alsa-info](static/images/audio/gather_audio_2.png)
```

Comment:
Can we add a statement about getting in touch with support if they are a customer?
If you own a System76 computer and require additional troubleshooting after going through these steps, [open a support ticket](https://system76.com/contact/support).